### PR TITLE
Corrected Issue w/ multiplicity matrix for microscopic MGXS data sets

### PR DESCRIPTION
**Pull request by nelsonag**
A user encountered a problem when creating microscopic MGXS data sets by nuclide. Specifically, the neutron balance was out of wack because the multiplicity matrix for a nuclide was being divided by the number density of the nuclide, instead of leaving the multiplicity matrix having values of 1 or greater.

The fix to this was done generically as there other Mgxs classes which also should not be divided by number density when creating a microscopic library. Specifically, a class attribute `_divide_by_density` is not consulted when the number density normalization is done.

The derived classes which have the `_divide_by_density` flag set to `False` are: `MultiplicityMatrixXs`, `ScatterProbabilityMatrix`, `Chi`, and `InverseVelocity`. The rest inherit the `Mgxs` base class' value of `True`.

### CODE REVIEW

File: openmc/mgxs/mgxs.py [hunk 1]
```diff
@@ -5257,7 +5271,7 @@ class Chi(MGXS):
     by_nuclide : bool
         If true, computes cross sections for each nuclide in domain
     domain : openmc.Material or openmc.Cell or openmc.Universe or openmc.RegularMesh
-        Domain for spatial homogenization
+        Domain for spatial homogenization`
```
Comment (paulromano) [inline openmc/mgxs/mgxs.py:5274]:
```suggestion
        Domain for spatial homogenization
```

Comment (nelsonag) [reply openmc/mgxs/mgxs.py]: !! Thank you! I looked at that when looking at my commit in gitk, and didnt see the apostrophe. Assumed I just deleted whitespace. 

Its fixed now. Also, I love the new ability to make changes right here in the github web interface, thats really cool.

**Comment by paulromano** [comment]
Looks good now; I'll merge after the 36 hour review period is up.

**Comment by nelsonag** [comment]
> 
> 
> I think 'beta' and 'decay-rate' in mdgxs.py also need the fix since they inherit their get_xs method from the MDGXS class.

Good point @GiudGiud, I will make that change right now

**Comment by nelsonag** [comment]
@GiudGiud I made the requested changes. For completeness, I also included ChiDelayed though its' get_xs method isnt subject to the same problem.

**Pull request merged**